pkg/build/builder/util.go

```diff
@@ -150,3 +154,15 @@ func ParseProxyURL(proxy string) (*url.URL, error) {
 
 	return proxyURL, err
 }
+
+// NameForBuildVolume returns a valid pod volume name for the provided build volume name.
```

Comment:
Did I see an exchange in a library-go PR that landed with *NOT* including a method like this there?

I ask because I presume (though I have not visited your PR there super recently) there is a need for such a method in openshift-controller-manager, and we'd ideally share the impl (albeit this is a simple thing).

But it may not be a given, and if so, say as much @coreydaley and we can resolve this thread ... but I'll mention it here anyway just in case.

So just let me know and we can go from there.

Reply:
we do have https://github.com/openshift/runtime-utils already for sharing build specific logic that could be a landing spot for such common methods that is specific to builds vs. library-go

Reply:
It was decided by the powers that be that this code should be duplicated between the repos instead of shared via library-go, and as such, the library-go pull request has been closed.

As for `runtime-utils`, all I see there is a `doc.go` in the build directory?

Reply:
> It was decided by the powers that be that this code should be duplicated between the repos instead of shared via library-go, and as such, the library-go pull request has been closed.
> 

ok so my recollection was correct in this case

> As for `runtime-utils`, all I see there is a `doc.go` in the build directory?

no there is also a pkg directory, with currently a "registries" subdir

technically speaking, per the readme, this repo was a common ground for https://github.com/containers related shared libs that build code would leverage, but there is no reason IMO would could not expand that a bit

@adambkaplan and @bparees are in the OWNER file

also note, that repo arose when similar utility functions were rejected by library-go